### ISSUE TITLE
Can't create new Design for RC2 project

### ISSUE THREAD

**Issue by alump** [vaadin/eclipse-plugin-legacy#777]
**Setup:**
- OS: MacOS 10.14.5
- Eclipse: 2019-06
- JDK: Corretto-11.0.3.7.1
- Vaadin Plugin: 4.1.0.beta4
- Vaadin Designer Plugin: 4.3.0.beta2

**Steps:**
1. Create new 14 project with Spring Boot base (with help of Vaadin add-on)
2. Run `mvn clean package` as required by designer add-on
3. Try to add new "platform" design

**Expected outcome:**
I can create new design

**Actual outcome:**
It still cries about me not running `mvn clean package` and missing npm dependencies.

<img width="933" alt="Screen Shot 2019-06-27 at 16 16 23" src="https://user-images.githubusercontent.com/1378660/60306866-f289ba80-98f6-11e9-9a61-9fef455335ce.png">

**Comment by alump** [comment]
Ok, user has to manually refresh the project structure, only then it will find the files `mvn clean package` created. You should mention this in guide, if you can't force refresh on project?

**Comment by Peppe** [comment]
@szolo is this fixed via the "run npm install automatically" work?

**Comment by szolo** [comment]
@Peppe yes, it's fixed. Plugin will automatically install dependencies and after that refresh the project structure.
